src/sentry/testutils/helpers/backups.py

```diff
@@ -465,7 +465,9 @@ def create_exhaustive_organization(
             monitor_type=AlertRuleMonitorType.ACTIVATED,
         )
         self.create_alert_rule_activation_condition(alert_rule=activated_alert)
-        self.create_alert_rule_activation(alert_rule=activated_alert, metric_value=100)
+        self.create_alert_rule_activation(
+            alert_rule=activated_alert, project=project, metric_value=100
```

Comment:
should this also have the query subscription id passed?

Reply:
Here it's not necessary since we default it in fixtures.py